apps/wallet/src/ui/app/helpers/computeGasBudget.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type CoinStruct } from '@mysten/sui.js';
+
+import { DEFAULT_GAS_BUDGET_FOR_PAY } from '_redux/slices/sui-objects/Coin';
+
+// This is a helper function to get a set of gas coins that can cover a given amount
+// It is from Coins.selectCoinSetWithCombinedBalanceGreaterThanOrEqual from sui.js
+// TODO: move this to sui.js
```

Comment:
Why not just do this? I don't know why we're not just defaulting to doing this

Reply:
done